packages/lite-insight/src/attribution/README.md

```diff
@@ -0,0 +1,7 @@
+## Instruction for code review
```

Comment:
If this description is only for cr, it can be put in the description of pr, if it is a new feature for LI, you want add some doc, you can add LI/ReadMe or ava-site doc

Reply:
This file has been removed.